money_honey_cr.py

```diff
@@ -0,0 +1,147 @@
+                else:
+                    print(output_string_start)
+                    for currency, currency_amount in converted_amounts.items():
+                        print("\t" + self.output_string_end(format(currency_amount, '>20.4f'), currency))
+            else:
+                output_json = {}
+
+                converted_currencies = {}
+                for currency, currency_amount in converted_amounts.items():
+                    converted_currencies[currency] = format(currency_amount, '.4f')
+
+                output_json['input'] = {'amount': format(amount, '.4f'), 'currency': input_currency}
+                output_json['output'] = converted_currencies
+                print("\n" + json.dumps(output_json, indent=4))
+
+    @staticmethod
+    def get_first_value(my_list):
+        return next(iter(my_list))
+
+    @staticmethod
+    def output_string_end(converted_amount, output_currency):
+        return str(converted_amount) + " " + output_currency
+
+    def check_currency(self, currency_changer, name, currency, symbol_currencies=None):
+        """
+        Checks if currency is valid and supported.
+        :param currency_changer: Instance of CurrenciesChanger.
+        :param name: Name of the input.
+        :param currency: Currency symbol or letter code.
+        :param symbol_currencies: Array of currency codes for specific symbol.
+        :return: Check function convert_symbol for returns.
+        """
+        if symbol_currencies is not None and currency.upper() in symbol_currencies:
+            return currency
+        elif symbol_currencies is None and currency_changer.is_supported_currency(currency.upper()):
+            return self.convert_symbol(currency_changer, name, currency)
+        else:
+            print("Incorrect " + name[2:] + ". Either it isn't supported or you messed up on your part.")
+            sys.exit(1)
+
+    def convert_symbol(self, currency_changer, input_name, symbol):
+        """
+        Converts all symbols to currency code and puts "--output_currency" into array.
+        :param currency_changer: Instance of class actual_currencies.CurrenciesChanger.
+        :param input_name: Name of the input. [--input_currency, --output_currency]
+        :param symbol: Symbol or currency code.
+        :return: Value if input_name equals "--input_currency", array otherwise.
+        """
+        if symbol in list(currency_changer.currency_signs.keys()):
+            currency_codes = currency_changer.currency_signs[symbol]
+            if input_name == "--input_currency" and len(currency_codes) > 1:
+                if input("Would you care to specify? y/n\n").lower() in ['y', 'yes']:
+
+                    print("You can choose from: [", end='')
+                    for i in range(len(currency_codes)):
+                        if i == len(currency_codes) - 1:
+                            print(currency_codes[i] + "]")
```

Comment:
tweaked out this part in [#7](https://github.com/ZlatyChlapec/MoneyHoney/commit/27c422c0ff89951c23ff16acb3dd78deae4de258).